### DISCUSSION TITLE
Multiple conditions

### DISCUSSION THREAD

**Discussion by DKhalil** [pestphp/pest#807] [Q&A] [answered]
Hello!

Is there a native pest way to test for multiple possible conditions? There's the `and()` function, is there something similar like `or()`?

**Comment by owenvoke** [comment, marked as answer]
I don't think this is something we natively have on Pest, however a suggestion is you could use the `->toMatchConstraint()` method with PHPUnit's `LogicalOr` constraint.

```php
use PHPUnit\Framework\Constraint;

it('can check with logical or', function () {
    expect('abc')->toMatchConstraint(
        Constraint\LogicalOr::fromConstraints(
            new Constraint\IsEqual('abc'),
            new Constraint\IsNull(),
        )
    );
});
```

It would be kind of nice to add this functionality in future though. 👀
